backend/calc.py

```diff
@@ -188,15 +267,22 @@ class Jobs(Enum):
     BLM = (115, Roles.CASTER, [])
     RDM = (115, Roles.CASTER, [Buffs.EMBOLDEN])
 
-    def __init__(self, job_mod, role, raidbuff):
+    def __init__(self, job_mod: int, role: Roles, raidbuff: list[Buffs]):
         self.job_mod = job_mod
         self.role = role
         self.raidbuff = raidbuff
 
 
 class Comp:
+    """
+    The party composition.
+
+    jobs: A list of all jobs in the party (can contain duplicates).
+    raidbuffs: All unique raidbuffs in the party.
+    n_roles: All unique roles in the party.
```

Comment:
Number of unique roles in the party (between 1-5). the docstring implies this is a set of roles but it's just an int.

Reply:
corrected